script/Amex-Select-Offers.js

```diff
@@ -90,22 +98,26 @@
           setTimeout(() => {
             addToCardButton.click();
             console.log(`Clicked "${offerName}" offer!`);
-          }, i * WAIT_TIME_MS);
+          }, WAIT_TIME_MS);
         } else {
           console.log(`Button for offer ${i} is not Add to Card, skipping...`);
         }
       } catch (e) {
         console.error(e);
       }
-    }
 
-    console.log("Done selecting offers!");
-    disableButton();
+      if (this.doneFlag) {
+        disableButton();
+        console.log("Done selecting offers!");
+      } else {
+        inProgressButton();
+      }
+    }
   }
 
   function createAndInsertButton() {
     button.classList.add(
-      "axp-offers__filter__filter___3yO4Q",
+      "offers-list",

```

Comment:
why are we removing the class header here?

Reply:
I did this for some reason I don't remember lol.